app/Kconfig

```diff
@@ -192,26 +192,6 @@ config SM_CMUX_UART_BUFFER_SIZE
 	  Size of the buffer for data received in CMUX mode.
 	  Same buffer size is used for both RX and TX.
 
-if SM_CMUX && SM_PPP
-
-config SM_MODEM_CELLULAR
```

Comment:
Add entries to migration_notes that these have been removed and what to do if there is replacement functionality.